### DISCUSSION TITLE
Question: trouble getting started with options picker from remote data source

### DISCUSSION THREAD

**Discussion by dgleba** [Budibase/budibase#12435] [Help]
**Checklist**
- I have looked at many pages and the university videos.
- maybe it is there but I don't see it.

**Hosting**

- Self
  "server": {
    "budibaseVersion": "2.13.15",
    "hosting": "docker-compose",
    "nodeVersion": "v18.18.2",
    "platform": "linux",
    "cpuArch": "x64",
    "cpuCores": 2,
    "cpuInfo": "Intel(R) Xeon(R) CPU E5-2640 v3 @ 2.60GHz",
    "totalMemory": "7.714141845703125GB",
  
**Describe the question**
I am struggling trying to populate an options picker from a remote data source.

This is what I am trying just in desperation:
Add data provider, form, and options picker.

![image](https://github.com/Budibase/budibase/assets/4622715/d9fd224d-6107-4ba7-84d7-1ad91c8dccf9)

But the field dropdown and the data provider dropdown are empty.
Maybe I don't understand how I should do it?

![image](https://github.com/Budibase/budibase/assets/4622715/4ab8c6f7-2fae-4bbe-8720-afcad2418a57)

![image](https://github.com/Budibase/budibase/assets/4622715/8bb60845-235d-410b-8796-aa1e42231f0e)

Can someone point me in the right direction?

**Comment by melohagan** [comment]
Hey @dgleba 

Thanks for the screenshots! Have you chosen a label and value column for your options picker?

**Comment by dgleba** [comment]
Thanks. That helped me make the dropdown populate.

I could not figure out how to save the selection to the database.

Is the following possible?

  In database A, there is a post table with city field and a tags field.
  Populate options picker with cities from database B, citylist table, column cityname
  Populate options picker with tags from database C, taglist table column tagname

Is that possible?
Is there a tutorial or document for what I want to do?

**Reply by ConorWebb96** [reply]
Hey @dgleba,

You need to select a field for it to save against. E.g. Cty if thats what it is called in your main table.
